src/core/util/vtypes.h

```diff
@@ -55,6 +55,14 @@
 #define INOUT
 #endif
 
+#if defined(DEFINED_DPCP_PATH_RX) && defined(DEFINED_DPCP_PATH_TX)
+#define DEFINED_DPCP_PATH_ONLY
+#endif
+
+#if defined(DEFINED_DPCP_PATH_RX) || defined(DEFINED_DPCP_PATH_TX)
+#define DEFINED_DPCP_PATH_ANY
+#endif
+
```

Comment:
What is a reason to do these in this place instead of `/dpcp.m4`?

Reply:
I didn't find a way to make conditional in m4. And i want DEFINED_DPCP_PATH_ANY not just be defined or not but be dependent on these.